nano/nano_node/entry.cpp

```diff
@@ -1494,7 +1494,10 @@ int main (int argc, char * const * argv)
 							bool error_or_pruned (false);
 							if (!state_block.hashables.previous.is_zero ())
 							{
-								prev_balance = node->ledger.balance_safe (transaction, state_block.hashables.previous, error_or_pruned);
+								if (auto balance = node->ledger.balance (transaction, state_block.hashables.previous))
+								{
+									prev_balance = balance.value ();
+								}
```

Comment:
Those `if (opt = ...) { value = opt.value () }` blocks could in many cases be replaced simply by one-liner: `opt.value_or({})` but it's a cosmetic improvement - new style is better either way.

Reply:
That's a good point. I'll convert these as I like that style better.